bitnami/nginx-ingress-controller/Chart.yaml

```
@@ -19,11 +19,12 @@ keywords:
   - web
   - www
   - reverse proxy
+kubeVersion: ">= 1.22"
```

Please, use single quotes
```suggestion
kubeVersion: '>= 1.22'
```